neqo-transport/src/recovery/mod.rs

```diff
@@ -863,6 +863,19 @@ impl LossRecovery {
         // pto_time to increase which might cause PTO for later packet number spaces to not fire.
         if let Some(pn_space) = pto_space {
             qtrace!([self], "PTO {}, probing {:?}", pn_space, allow_probes);
+            // If we hit a PTO while we don't have a largest_acked yet, also do a congestion control
+            // reaction (because otherwise none would happen).
+            if let Some(space) = self.spaces.get(pn_space) {
+                if space.largest_acked.is_none() {
+                    path.borrow_mut().on_packets_lost(
```

Comment:
Do we really want to mark specific packets lost or do we want to just tell the path to reduce CWND directly?  That is, via something like `on_congestion_event()`.

Reply:
I tried that, but it seemed to need a bunch of additional code changes, since that is not currently exposed. Can give it another go.

Reply:
@martinthomson take another look; I now expose `on_congestion_event`.